web_src/js/features/repo-home.js

```diff
@@ -64,10 +66,11 @@ export function initRepoTopicBar() {
         topicPrompts.formatPrompt = responseData.message;
 
         const {invalidTopics} = responseData;
-        const $topicLabels = $topicDropdown.children('a.ui.label');
+        const topicLabels = topicDropdown.querySelectorAll(':scope > a.ui.label');
```

Comment:
I think it could simply use `topicDropdown.querySelectorAll('a.ui.label')`, because other places just use it (without scope)

Reply:
https://github.com/go-gitea/gitea/pull/30191/commits/2cc10ac988defedf20a5d86261ad7ebdb1019ce5